tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+"""
+.. _tut_erp:
+
+Event Related Potentials (ERPs) Measures
+==================================================
+
```

Comment:
can you add some text here to explain what you will explain below. I kind of abstract.

Also if you have some references that have used what you propose here it would be great.

Reply:
Will do. I cannot figure out how to add references. The plan is to cite Luck (2014) for some of this.

Reply:
add BibTex ref to `doc/references.bib` then use `:footcite:` where you need it and `.. footbibliography::` at the end like in other tutorials, e.g.

https://github.com/mne-tools/mne-python/blob/master/tutorials/preprocessing/plot_45_projectors_background.py#L233

Reply:
Thanks @larsoner